tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -114,4 +127,16 @@ Iterative Image Test
     Clean Up Server
     Stop JupyterLab Notebook Server
     Go To    ${ODH_DASHBOARD_URL}
-    Sleep    10    reason=Waiting for ODH Dashboard home page to load
\ No newline at end of file
+    Sleep    10    reason=Waiting for ODH Dashboard home page to load
+
+Disable Access To Port Forwarding And Close Browser
+    [Documentation]  Disables Access to Port Forwarding and Closes All Browsers
+    [Arguments]     ${PROC}
```

Comment:
Keyword parameters should not be in upper-case. It could be renamed to ${port_forwarding_process} to be more meaningful

Reply:
Updated.